src/views/home/UserInput.js

```diff
@@ -76,14 +76,36 @@ export default class UserInput extends Component {
     e.preventDefault();
     this.setState(prevState => {
       let ticket = { ...prevState.ticket };
-      ticket.screenshots = ticket.screenshots.filter((t, index) => i !== index);
+      ticket.screenshots = ticket.screenshots.filter((s, index) => i !== index);
       return { ticket };
     });
   };
 
+  cleanScreenshots = ticket => {
+    const isValidScreenshot = s => {
+      return typeof s.height === 'number' && typeof s.width === 'number';
```

Comment:
This will let floating point numbers through.

Reply:
actually i tested it and because of how the input is handled a user can't even try to submit a floating point number. I tried copying and pasting a floating point number as well and the frontend auto converted it to an int inside of the input